frontend/src/Components/Map/InfoCard.tsx

```diff
@@ -50,7 +50,14 @@ const InfoCard: React.FC<Props> = ({ hidden, roadData, surveyData }) => {
           </div>
           <div className="roadinfo-card-text">
             <span className="text-title"> Branch Number: </span>
-            <span className="card-road-length">{roadData.branches.length}</span>
+            <span className="card-road-length">
+              {' '}
+              {roadData.branches.reduce(
+                (totalLength, branch) => totalLength + branch.length,
+                0,
+              )}{' '}
+              {/* roadData.branches is a 2D array */}
+            </span>
```

Comment:
`branch` is a string so i don't get why you do `branch.length`...

Reply:
umm, yea, I thought wrong before, should keep the original one